examples/cc/lib/BUILD

```diff
@@ -1,9 +1,19 @@
 cc_library(
-    name = "lib",
+    name = "actual_lib",
     srcs = [
         "lib.c",
         "private.h",
     ],
+)
+
+cc_library(
+    name = "headers_only",
```

Comment:
maybe impl_lib and header_only_lib?

(then you avoid the name collision of "actual = headers_only" below)

Reply:
The alias is on purpose to prove that they don't mess with things. I'll rename these though in a followup PR (auto-merged).